src/main/java/net/mcreator/workspace/elements/ModElementManager.java

```diff
@@ -61,7 +61,7 @@ public ModElementManager(@Nonnull Workspace workspace) {
 		this.gsonAdapter = new GeneratableElement.GSONAdapter(this.workspace);
 
 		GsonBuilder gsonBuilder = new GsonBuilder().registerTypeHierarchyAdapter(GeneratableElement.class,
-				this.gsonAdapter).disableHtmlEscaping().setPrettyPrinting().setLenient();
+				this.gsonAdapter).disableHtmlEscaping().serializeNulls().setPrettyPrinting().setLenient();

```

Comment:
This is not good as it will increase mod elements considerably where not needed

Reply:
Should I move `customProperties` from `Map<String, Procedure>` to like `List<Tuple<String, Procedure>>` to avoid that?

Reply:
So the problem is procedures can be null? In what cases does this happen?

Reply:
If someone creates a property and keeps its value procedure on default, `(zero)`.

Reply:
But does it make sense to allow that? Fixed property is no property I would say

Reply:
Should I validate procedure selectors?

Reply:
If there is not use of constant property, then I would validate them yes and keep Map and revert null serialization

Reply:
>I could try.

Reply:
✔️ Fixed